article/_posts/2021-03-26-QR_DQN.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: review
+title: "Distributional Reinforcement Learning with Quantile Regression"
+tags: reinforcement
+author: "Antoine Théberge"
+cite:
+    authors: "Will Dabney, Mark Rowland, Marc G. Bellemare, Rémi Munos"
+    title:   "Distributional Reinforcement Learning with Quantile Regression"
+    venue:   "AAAI 2018"
+pdf: "https://arxiv.org/pdf/1710.10044.pdf"
+---
+
+
+# Highlights
+
+- Closes the gap between the theory and practice on distributional reinforcement learning
+- Achieves state-of-the-art results of the Atari 2600 suite of games
+
+# Introduction
+
+[Distributional Reinforcement Learning](https://vitalab.github.io/article/2021/02/12/C51.html) has been proposed as an alternative to standard reinforcement learning where the whole distribution of future rewards is learned instead of its expectation:
+
+$$Z(x,a) \stackrel{D}{=} R(x,a) + \gamma Z(X', A'),$$
+
+where $$Z(x,a) \in \mathbb{R}^{\vert A \vert \times N}$$ a set of atoms forming the [probability mass function (PMF)](https://en.wikipedia.org/wiki/Probability_mass_function) of the return distribution. The goal then became to minimize the distance between $$Z_\theta(x,a)$$ and $$R(x,a) + \gamma Z_\theta(X',A')$$. While the [Wasserstein metric](https://en.wikipedia.org/wiki/Wasserstein_metric) could be used in theory to act as a distance metric between the two distributions, it could not in practice be used as a loss function to be minimized by stochastic gradient descent.
```

Comment:
The goal then became : The goal then becomes ??

Reply:
"it could not in practice be used (...)". Does this refer at the time when the cited paper introduced the Distributional Reinforcement Learning concept, or else is it still the case? It is unclear. And the `The Wasserstein distance` subsection does not clarify it either IMO.

Reply:
It was indeed unclear which paper I was referring to. I have reformulated the section in 2c83cb1